components/List/List.View.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { FlatList, SafeAreaView, StatusBar, StyleSheet } from "react-native";
+import Header from "./Header.View";
+import Item from "./Item.View";
+
+const styles = StyleSheet.create({
```

Comment:
One of the patterns we encourage is splitting out these stylesheets into their own component specific .style.js file. As a component grows they can become pretty heavy and make the view layer difficult to parse if left here.  Putting each component inside its own folder of the same name, and then putting all the component specific files inside that folder has helped us keep those organized